bundles/azure-bundle/build.gradle.kts

```diff
@@ -56,7 +40,8 @@ tasks.withType(ShadowJar::class.java) {
   relocate("org.apache.httpcomponents", "org.apache.gravitino.azure.shaded.org.apache.httpcomponents")
   relocate("org.apache.commons", "org.apache.gravitino.azure.shaded.org.apache.commons")
   relocate("com.fasterxml", "org.apache.gravitino.azure.shaded.com.fasterxml")
-  relocate("com.google.guava", "org.apache.gravitino.azure.shaded.com.google.guava")
+  relocate("com.google.common", "org.apache.gravitino.azure.shaded.com.google.common")
+  relocate("org.eclipse.jetty", "org.apache.gravitino.gcp.shaded.org.eclipse.jetty")
```

Comment:
Why it is gcp here?

Reply:
Sorry, it's a mistake and I will correct it.

Reply:
Please carefully go through the code before calling out for review (to save reviewer's time).